community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -41,7 +41,6 @@
     NodeState,
     chunked,
     dirs,
```

Comment:
Remove the import for `parse_gcp_timestamp` but I see it still reference in L471

Reply:
Thank you for pointing this out! Pylance fooled me this time.
It makes me sad that our tests can't catch even such issue.